ubyssey/templates/about/header.html

```diff
@@ -0,0 +1,58 @@
+{% load staticfiles %}
+<header class="header">
+  <style type="text/css">
+
+
+
+    
+  </style>
+  <div class="cont">
+      <div class="c-header__logo">
+      <img class="c-header__logo__image" src="{% static 'images/a/ubyssey-logo.svg' %}"/>
+      <span class="c-header__logo__text">About</span>
+    </div>
+    <div id="header-main-cont">
+      <h2 class="c-header__hook">The definitive source of news on campus.</h2>
+      <div>
+        <svg id="broadcast-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 139.68 168.19">
```

Comment:
:100: 
Have you considered saving the style within svg file like in the second part of [this answer](https://stackoverflow.com/a/20720935/5059325)? This inline totally works, but external svg file is favorable if possible so that browser will cache it on the client side

Sidenote: if we're using inline svg we can remove the svg files included in this PR

Reply:
Yea for sure – the flash animation for the header actually directly manipulates certain elements in the inline svg element, so I'll just remove the external file